tests/model_test.py

```diff
@@ -8,25 +8,20 @@
     [
         ("attentive_lstm", True, models.AttentiveLSTMEncoderDecoder),
         ("attentive_lstm", False, models.AttentiveLSTMEncoderDecoder),
-        (
-            "feature_invariant_transformer",
-            True,
-            models.FeatureInvariantTransformerEncoderDecoder,
-        ),
         ("lstm", True, models.LSTMEncoderDecoder),
         ("lstm", False, models.LSTMEncoderDecoder),
         (
             "pointer_generator_lstm",
             True,
-            models.PointerGeneratorLSTMEncoderDecoderFeatures,
+            models.PointerGeneratorLSTMEncoderDecoder,
         ),
         (
             "pointer_generator_lstm",
             False,
-            models.PointerGeneratorLSTMEncoderDecoderNoFeatures,
+            models.PointerGeneratorLSTMEncoderDecoder,
```

Comment:
That you can make the change you made here is proof of the positive impact of this PR ;) 

(no work required obviously)